lesson1/serhii_khrychov_hometask.rb

```diff
@@ -0,0 +1,24 @@
+class Foo
+  def initialize(uniq = :not_uniq)
+    @@instance_num ||= 0
+    @@instance_num += 1
+    @instance_num = @@instance_num
+    @uniq = uniq
+  end
+
+  def my_instance
+    return ":uniq" if @uniq == :uniq
```

Comment:
Interesting decision. I thought you would use singleton method for `:uniq` instance. But your solution also works.